exonum-java-binding/core/pom.xml

```diff
@@ -312,17 +315,31 @@
           <!-- Compile protobuf description of Java service runtime deploy arguments
                to Python for exonum_java_runtime_plugin -->
           <execution>
-            <id>protobuf-compile-python</id>
+            <id>protobuf-compile-python-deploy-arguments</id>
             <goals>
               <goal>compile-python</goal>
             </goals>
             <configuration>
-              <outputDirectory>${protobuf.python.outputDirectory}</outputDirectory>
+              <outputDirectory>${protobuf.python.deployArguments.outputDirectory}</outputDirectory>
               <includes>
                 <include>deploy_arguments.proto</include>
               </includes>
             </configuration>
           </execution>
+          <!-- Compile protobuf description of Java service runtime deploy arguments
```

Comment:
Can't they share the directory for proto sources?

Reply:
Unfortunately, it will force as to change packages structure, which leads to changes in plugins FQNs (in configs), and looks a bit ugly. Technically the plugins are separated packages with separated dependencies, so I'd leave it as it is.